neo.UnitTests/UT_RemoteNode.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Akka.IO;
+using Akka.TestKit.Xunit2;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Network.P2P;
+using Neo.Network.P2P.Capabilities;
+using Neo.Network.P2P.Payloads;
+using System;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_RemoteNode : TestKit
+    {
+        private static readonly Random TestRandom = new Random(1337); // use fixed seed for guaranteed determinism
+
+        private static NeoSystem testBlockchain;
+
+        public UT_RemoteNode()
+            : base($"remote-node-mailbox {{ mailbox-type: \"{typeof(RemoteNodeMailbox).AssemblyQualifiedName}\" }}" +
+                $"protocol-handler-mailbox {{ mailbox-type: \"{typeof(ProtocolHandlerMailbox).AssemblyQualifiedName}\" }}")
+        {
+        }
+
+        [ClassInitialize]
+        public static void TestSetup(TestContext ctx)
```

Comment:
Never used parameter?

Reply:
Removed.

Reply:
This is required for the UT Culture:
``` CSharp
 var testContext = new object[] { new UnitTestContext() };
```  
This was added because UT Culture was failing, complaining that this method required a TestContext parameter